### DISCUSSION TITLE
My Github account has been suspended and I'm not getting a supportive response

### DISCUSSION THREAD

**Discussion by tongyao-zhang** [community/community#124641] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

The previous account could not log in due to f2a problem, so I registered a new one. When I logged in, my github account was found to be disabled. I read the manual rules of github, but I did not find any violation, so I suspended my account. I don't know where the violation is, if you know, please tell me, lift the suspension, modify the corresponding violation, thank you very much!

I have submitted the case ID for many times, but I have not received any reply, including the mail. My case ID is as follows:#2718700 ,The ID problem of this case has not been notified by email for 2 months!

My account: zhang-tongyao corresponding email: 1973689961@qq.com

**Comment by tongyao-zhang** [comment]
Who can help me? If so, there is the contact information of github administrator, please @github administrator to help me solve it. Thank you very much again!

**Reply by 0xiinit** [reply]
myb its new

**Comment by ebndev** [comment, marked as answer]
Hi @tongyao-zhang,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
